src/pages/api/reviewed-preprints.page.ts

```diff
@@ -19,7 +19,7 @@ type ReviewedPreprintItemResponse = {
 type ReviewedPreprintSnippet = {
   id: string,
   doi: string,
-  pdf: string,
+  pdf?: string,
```

Comment:
@nlisgo This is/was a little concerning - do you foresee any consequences of this for API / Journal?

Reply:
Thanks, glad it was just marked wrong here 👍